service/telemetry/config.go

```diff
@@ -103,3 +104,12 @@ type MetricsConfig struct {
 	// Address is the [address]:port that metrics exposition should be bound to.
 	Address string `mapstructure:"address"`
 }
+
+// TracesConfig defines settings to configure context propagation of the collector traces. Collector uses
+// go.opentelemetry.io/contrib/propagators/autoprop package to propagate the context and OTEL_PROPAGATORS
+// environment variable can be set to configure the propagators.
```

Comment:
We should have a more general description here
```suggestion
// TracesConfig defines the configurable settings for service telemetry traces.
```

The other details are implementation details that I don't think we should document, since it may get out of sync if we decide to use a different propagator.